mteb/tasks/Classification/ara/AJGT.py

```diff
@@ -37,3 +40,14 @@ class AJGT(AbsTaskClassification):
 }
 """,
     )
+
+    def dataset_transform(self):
+        self.dataset["test"] = self.dataset["train"]
```

Comment:
Won't this the overwritten in the following (if not it should ensure 100&% leakage)
```suggestion
```